trio/_threads.py

```diff
@@ -8,119 +8,29 @@
 import trio
 
 from ._sync import CapacityLimiter
-from ._core import enable_ki_protection, disable_ki_protection, RunVar
+from ._core import enable_ki_protection, disable_ki_protection, RunVar, _entry_queue
```

Comment:
There's a weird rule we follow: any code outside of the `_core/` directory is not allowed to "reach inside" the `_core` namespace to access private attributes. (Meaning, attributes with leading underscores.)

The rationale is: we want to make sure that if someone needs to implement something like `from_thread.*` in a third party library on top of trio, then that should be possible without using any private APIs. Everything in `_core`'s public API is re-exported as part of `trio`'s public API, so if we can implement our own stuff using only `_core`'s public API, it means other people will be able to do the same. There's a bit more discussion here: https://trio.readthedocs.io/en/latest/design.html#a-brief-tour-of-trio-s-internals

In this case, that means we can't import `_entry_queue`. But, it looks like the only thing we use it for is to look up the `TrioToken` class object, and that's already exported from `_core`, so we can just do `from ._core import TrioToken` directly.